Resources/Prototypes/_Harmony/Entities/Markers/Spawners/ghost_roles.yml

```
@@ -0,0 +1,22 @@
+﻿- type: entity
+  id: SpawnPointGhostGizard
+  name: ghost role spawn point
+  suffix: Gizard
+  parent: MarkerBase
```

The order is incorrect
```suggestion
  parent: MarkerBase
  id: SpawnPointGhostGizard
  name: ghost role spawn point
  suffix: Gizard
```